### PR TITLE
Add helm chart description to README

### PR DESCRIPTION
**Pull request by kota2and3kan**
## Description

This PR adds the description of Helm Chart for `Scalar Admin for Kubernetes`.

Please take a look!

## Related issues and/or PRs

N/A

## Changes made

* Update README.

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

### CODE REVIEW

**Review by josh-wong — CHANGES_REQUESTED** [review]
I've added one suggestion with a couple questions. PTAL!

File: README.md [hunk 1]
```diff
@@ -111,4 +111,4 @@ The `scalar-admin-for-kubernetes` CLI tool executes Kubernetes APIs in its inter
 
 ## Run the CLI tool in a Kubernetes environment by using a Helm Chart
 
-Coming soon.
+You can use Helm Chart to deploy Scalar Admin for Kubernetes in the Kubernetes environment. See [Configure a custom values file for ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/helm-charts/configure-custom-values-scalardb-cluster/) and [How to deploy Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/how-to-deploy-scalar-admin-for-kubernetes/) for more details on Helm Chart of Scalar Admin for Kubernetes.
```
[the remark this answers is not in the document]
Comment (kota2and3kan) [reply README.md:114]: @josh-wong 
Thank you for your review!

> - How about if we combine the two sentences into one sentence?

Looks good! Thank you for your suggestion!

> - Do we need to include links to any other docs for configuring custom values files?
>   - For example, we have other docs that describe how to configure a custom values file listed at [Configure a custom values file for Scalar Helm Charts](https://scalardb.scalar-labs.com/docs/latest/helm-charts/configure-custom-values-file/). If so, I think we'll need to reword this a bit more.

Ah, sorry... I set the wrong document link...
For Scalar Admin for Kubernetes, users only need to see the following two documents.

1. [Configure a custom values file for Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/configure-custom-values-scalar-admin-for-kubernetes/)
1. [How to deploy Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/how-to-deploy-scalar-admin-for-kubernetes/)

And, basically, they don't need to update other custom values files like ScalarDB Cluster or ScalarDL to run Scalar Admin for Kubernetes.

So, how about the following sentence?

```suggestion
For details on how to use a Helm Chart to deploy Scalar Admin for Kubernetes in a Kubernetes environment, see:

1. [Configure a custom values file for Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/configure-custom-values-scalar-admin-for-kubernetes/)
1. [How to deploy Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/how-to-deploy-scalar-admin-for-kubernetes/)
```

[the remark this answers is not in the document]
Comment (josh-wong) [reply README.md:114]: Thank you for the explanation! That makes sense to me😄

[the remark this answers is not in the document]
Comment (josh-wong) [reply README.md:114]: Actually, should we point to both the ScalarDB and ScalarDL docs sites in this case?

```suggestion
For details on how to use a Helm Chart to deploy Scalar Admin for Kubernetes in a Kubernetes environment, see the following documentation based on the product you're using:

### ScalarDB

1. [Configure a custom values file for Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/configure-custom-values-scalar-admin-for-kubernetes/)
1. [How to deploy Scalar Admin for Kubernetes](https://scalardb.scalar-labs.com/docs/latest/helm-charts/how-to-deploy-scalar-admin-for-kubernetes/)

### ScalarDL

1. [Configure a custom values file for Scalar Admin for Kubernetes](https://scalardl.scalar-labs.com/docs/latest/helm-charts/configure-custom-values-scalar-admin-for-kubernetes/)
1. [How to deploy Scalar Admin for Kubernetes](https://scalardl.scalar-labs.com/docs/latest/helm-charts/how-to-deploy-scalar-admin-for-kubernetes/)
```

[the remark this answers is not in the document]
Comment (kota2and3kan) [reply README.md:114]: > Actually, should we point to both the ScalarDB and ScalarDL docs sites in this case?

Ah, I see. The contents are the same, but the URLs are different.
Thank you for your suggestion!

**Review by josh-wong — APPROVED** [review]
LGTM! Thank you!🙇‍♂️

**Review by supl — APPROVED** [review]
LGTM.
Thank you 🙇

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**